src/components/Navigation.tsx

```diff
@@ -1,109 +1,153 @@
 import React from 'react';
 
-import { Flex, Link } from '@chakra-ui/react';
+import { HamburgerIcon, CloseIcon } from '@chakra-ui/icons';
+import {
+  Box,
+  Drawer,
+  DrawerBody,
+  DrawerContent,
+  DrawerHeader,
+  DrawerOverlay,
+  Flex,
+  Link,
+  useDisclosure,
+} from '@chakra-ui/react';
 import { Link as GatsbyLink } from 'gatsby';
+import Img from 'gatsby-image';
 
-function Navigation(): JSX.Element {
+interface NavLinkProps {
+  link: string;
+  text: string;
+  onClick?: () => void;
+}
+
+interface NavigationProps {
+  data: GatsbyTypes.LayoutQuery;
+}
+
+function NavLink({ link, text }: NavLinkProps): JSX.Element {
   return (
-    <Flex
-      h="55px"
-      wrap="nowrap"
-      mx="auto"
-      flexDir="row"
-      w="full"
-      justifyContent="space-evenly"
-      alignItems="center"
+    <Link
+      _hover={{
```

Comment:
Could these styles (and the styles for `MobileNavLink` be migrated to `theme.ts`? The components seem like they're just wrappers for `Link`

Reply:
Created a styling for `NavLink` in `theme.ts`. It currently uses `useStyleConfig` to apply the styles in our custom component.

Using variant in `Link` by itself was not helpful considering different styles for desktop/mobile view and we cannot have multiple variants.